ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentAssignment/EquipmentAssignmentItemModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ElectronicObserverTypes;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.EquipmentAssignment;
+
+public class EquipmentAssignmentItemModel
+{
+	public int Id { get; set; }
+
+	public EquipmentUpgradePlanItemModel Plan { get; set; } = null!;
```

Comment:
Make it a required property so you don't need the `= null!;` part.
`public required EquipmentUpgradePlanItemModel Plan { get; set; }`